### ISSUE TITLE
Select: WCAG issues

### ISSUE THREAD

**Issue by kristiina-richards** [TEDI-Design-System/react#348]
### Description of the problem

1. JAWS doesn't read out the selected value correctly when moving around the open select options. It is shown visually but it doesn't capture it
**Requirement:** [link](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html)
2. With a value selected and focus out of the select box, the delete button contrast is too low. (1.6:1) 
**Requirement:** [link](https://www.w3.org/WAI/WCAG22/Understanding/non-text-contrast.html)
3. Not possible to move to the delete button with a keyboard. Value can only be deleted with backspace
**Requirement:** [link](https://www.w3.org/WAI/WCAG22/Understanding/keyboard.html)
4. JAWS doesn't move to the delete button
**Requirement:** [link](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value.html)
5. JAWS doesn't read required select items as required. Seems as the required prop isn't passed onto the input
**Requirement:** [link](https://www.w3.org/WAI/WCAG22/Understanding/info-and-relationships.html)
